amd_openvx/openvx/api/vx_api.cpp

```diff
@@ -9801,6 +9801,17 @@ VX_API_ENTRY vx_status VX_API_CALL vxQueryTensor(vx_tensor tensor, vx_enum attri
 
 #endif
 #endif
+			case VX_TENSOR_BUFFER_HOST:
+                if (size == sizeof(vx_uint8 *)) {
+                    if (data->buffer) {
+                        *(vx_uint8 **)ptr = data->buffer;
+                    }
+                    else {
+                        *(vx_uint8 **)ptr = NULL;
+                    }
+                    status = VX_SUCCESS;
+                }
+                break;
```

Comment:
We discussed to change this to Void pointer and handle it ryt?

Reply:
The change has been added it is not void * we are checking if it is a unit8_t* only